scripts/clean_pecd_data.py

```diff
@@ -127,16 +149,24 @@ def read_pecd_file(
     )
 
     with mp.Pool(processes=snakemake.threads) as pool:
-        demand = list(tqdm(pool.imap(func, nodes), **tqdm_kwargs))
+        pecd = list(tqdm(pool.imap(func, nodes), **tqdm_kwargs))
 
+    if all(data is None for data in pecd):
+        raise ValueError(
+            f"No PECD data found for {pecd_tech} in {pyear}. Please specify a technology covered within the TYNDP PECD data."
+        )
+    pecd_df = pd.concat(pecd, axis=1)
```

Comment:
i still remember concat being unhappy about `None`. Is it impossible that any of the read_pecd_file calls returns None? 

Maybe use `if any(...)` above or change above to

```python
pecd = [x for x in tqdm(pool.imap(func, nodes), **tqdm_kwargs) if x is not None]
```
and then check for an empty list instead of the `all` call?